src/components/common/Stepper.vue

```diff
@@ -25,7 +25,7 @@
           <v-icon class="step__btn2" size="30" color="success darken-1" v-show="isValid(step.to)">
             mdi-check-circle
           </v-icon>
-          <v-icon class="step__btn2" size="30" color="error darken-1" v-show="!isValid(step.to) && getValidateSteps">
+          <v-icon class="step__btn2" size="30" color="error darken-1" v-show="!isValid(step.to) && getShowErrors">
```

Comment:
`getValidateSteps` becomes True when the user has gone to the review page.

`getShowErrors` becomes True when the user tries to file (I think).

These don't mean the same thing.

Instead of this, I'm thinking something is missing to make the subject steps invalid.

Reply:
`getValidateSteps` becomes True after the user tries to file,
 `getShowErrors` becomes True after the user visits the review page.

The step styling uses the same validation, for isInvalid.
https://github.com/bcgov/business-create-ui/blob/a1660d9ed66215ff35b2a19152d144cd3684b781/src/components/Dissolution/CompleteResolution.vue#L393

Reply:
OK, I was wrong. Thanks for the references.

So, why is this broken only now and for only certain steps? It should have been broken for all steps up to now, right?

Reply:
Resolving this comment. I need to understand what the problem is.

Reply:
I think it was broken for all the steps, at least it was when I checked. I just think the ticket didn't specifically mention all the steps. Not really sure why.